src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -379,7 +379,7 @@ function BaseSelectionList<TItem extends User | RadioItem>(
     });
 
     /** Calls confirm action when pressing CTRL (CMD) + Enter */
-    useKeyboardShortcut(CONST.KEYBOARD_SHORTCUTS.CTRL_ENTER, onConfirm, {
+    useKeyboardShortcut(CONST.KEYBOARD_SHORTCUTS.CTRL_ENTER, onConfirm ?? selectFocusedOption, {
         captureOnInputs: true,
         shouldBubble: !flattenedSections.allOptions[focusedIndex],
         isActive: !disableKeyboardShortcuts && !!onConfirm && isFocused,
```

Comment:
Now that we fall back to `selectFocusedOption`, you may want to remove the `!!onConfirm` check here as it breaks the logic a bit.